satpy/readers/seviri_l1b_native.py

```diff
@@ -511,6 +517,81 @@ def _get_calib_coefs(self, channel_name):
             radiance_type=radiance_types[band_idx]
         )
 
+    def _add_scanline_acq_time(self, dataset, dataset_id):
+        """Add scanline acquisition time to the given dataset."""
+        if dataset_id['name'] == 'HRV':
+            tline = self._get_acq_time_hrv()
+        else:
+            tline = self._get_acq_time_visir(dataset_id)
+        acq_time = get_cds_time(days=tline['Days'], msecs=tline['Milliseconds'])
+        add_scanline_acq_time(dataset, acq_time)
+
+    def _get_acq_time_hrv(self):
+        """Get raw acquisition time for HRV channel."""
+        tline = self.dask_array['hrv']['acq_time']
+        tline0 = tline[:, 0]
+        tline1 = tline[:, 1]
+        tline2 = tline[:, 2]
+        return da.stack((tline0, tline1, tline2), axis=1).reshape(
+            self.mda['hrv_number_of_lines']).compute()
+
+    def _get_acq_time_visir(self, dataset_id):
+        """Get raw acquisition time for VIS/IR channels."""
+        # Check if there is only 1 channel in the list as a change
+        # is needed in the arrray assignment ie channl id is not present
+        if len(self.mda['channel_list']) == 1:
+            return self.dask_array['visir']['acq_time']
```

Comment:
The other two `_get_acq_time*` returns are computed, but this one isn't, is this intended?

Reply:
Oh, good catch! That should indeed be computed as well!